mscore/inspector/inspector.cpp

```diff
@@ -547,7 +549,8 @@ InspectorVBox::InspectorVBox(QWidget* parent)
             { Pid::BOTTOM_MARGIN, 0, vb.bottomMargin, vb.resetBottomMargin },
             { Pid::BOX_HEIGHT,    0, vb.height,       0                    }
             };
-      mapSignals();
+      const std::vector<InspectorPanel> ppList = { { vb.title, vb.panel } };
+      mapSignals(iiList, ppList);
```

Comment:
These changes may also have unwanted side effects since: with the old implementation `iList` here overwrote the one from `InspectorElementBase`, but the new implementation makes all the properties from `InspectorElementBase` be processed by `mapSignals()` even though the corresponding controls for them (the panel with "visible", "autoplace" checkboxes etc.) are not displayed.

Actually I wouldn't call the current design of `mapSignals()` good but we should either restructure this function to enable its usage like is presented in this pull request (which might be not very useful right now in spite of the upcoming inspector redesign) or follow the old pattern of its usage, with that requirement to call `mapSignals()` only once and in the most descendant class, overwriting properties set up in base class constructor and other its features.

Reply:
So you'd suggest we don't create new variables but rather overwrite `iList` and `pList`?

And don't worry, the abstract inspectors model from the new inspector takes very good care of the properties and doesn't have confusing things like `mapSignals()`.

Reply:
Actually, I think what you called "unwanted side effects" are necessary. `mapSignals()` shouldn't be called in `InspectorElementBase`, so it is the final classes' responsibility to do it for the parent class. The "Element" panel is not visible for the final classes, but `mapSignals()` handles it in them.

Reply:
But for those inspectors which do not contain "Element" panel handling properties of `InspectorElementBase` doesn't seem to make sense. What is this handling do you consider necessary for?

Reply:
Yes, you're right, I changed the code.